qsimcirq/qsim_simulator.py

```diff
@@ -51,16 +53,124 @@ def __init__(self,
                        final_simulator_state=final_simulator_state)
 
 
-class QSimSimulator(SimulatesAmplitudes, SimulatesFinalState):
+class QSimSimulator(SimulatesSamples, SimulatesAmplitudes, SimulatesFinalState):
 
-  def __init__(self, qsim_options: dict = {}):
+  def __init__(self, qsim_options: dict = {},
+               seed: value.RANDOM_STATE_OR_SEED_LIKE = None):
     if any(k in qsim_options for k in ('c', 'i')):
       raise ValueError(
           'Keys "c" & "i" are reserved for internal use and cannot be used in QSimCircuit instantiation.'
```

Comment:
Should "t" and "v" be added here?

Reply:
Done.

Reply:
Reverted this change - the "t" and "v" values here are only default values. Users can set them manually.